nameko/standalone/rpc.py

```diff
@@ -66,12 +67,17 @@ class PollingQueueConsumer(object):
     """
     def __init__(self, timeout=None):
         self.timeout = timeout
+        self.replies = {}
 
     def _setup_queue(self):
-        self.channel = self.connection.channel()
+        channel = self.connection.channel()
         # queue.bind returns a bound copy
-        self.queue = self.queue.bind(self.channel)
-        maybe_declare(self.queue, self.channel)
+        self.queue = self.queue.bind(channel)
+        maybe_declare(self.queue, channel)
+        consumer = Consumer(channel, queues=[self.queue], no_ack=False)
+        consumer.callbacks = [self.on_message]
+        consumer.consume()
+        self.consumer = consumer
```

Comment:
What happens to the first consumer in the event of a timeout, temporary error or ``KeyboardInterrupt``? Can we end up multiple consumers?